README.md

```diff
@@ -200,7 +200,63 @@ published on Hackage, or part of Stackage snapshots, using Bazel.
 
 [hazel]: https://github.com/formationai/hazel
 
-## Troubleshooting `rules_haskell` development
+## Troubleshooting
+
+### `as`: no such file or directory
+
+If you see error messages complaining about missing `as` executable:
```

Comment:
Or `ld', if you like, or anything really (I had to put `touch` in there as well for `language-c` to compile - I don't know why).